WebApplication1/Services/GameStatusService.cs

```diff
@@ -72,7 +72,9 @@ private LiveMatch GetLiveMatchInfo(ServerInfo serverInfo, PlayerDetailShack[] pl
             var result = new LiveMatch()
             {
                 MapLabel = serverInfo.MapLabel,
+                MapName = serverInfo.MapName,
```

Comment:
The return of class serverInfo, has no member _MapName_, could you remove or revise it?

You can check the **serverinfo** command return here:
https://pavlovwiki.com/index.php/Rcon_Overview_and_Commands